x-pack/plugins/security_solution/common/api/detection_engine/rule_management/export_rules/export_rules_route.gen.ts

```diff
@@ -19,13 +20,7 @@ export const ExportRulesRequestQuery = z.object({
   /**
    * Determines whether a summary of the exported rules is returned.
    */
-  exclude_export_details: z.preprocess(
-    (value: unknown) => (typeof value === 'boolean' ? String(value) : value),
-    z
-      .enum(['true', 'false'])
-      .default('false')
-      .transform((value) => value === 'true')
-  ),
+  exclude_export_details: BooleanFromString.optional().default(false),
```

Comment:
So, just to understand what happened here:

You created the `BooleanFromString` helper in this same PR, but the schema file for `export_rules_route` wasn't updated in any way.
So how did this get updated? Did you do it manually?

Reply:
I've updated the code generator for booleans and arrays in request queries. You can find the changes in the template file at `packages/kbn-openapi-generator/src/template_service/templates/zod_query_item.handlebars`.